src/worker.cc

```diff
@@ -1497,6 +1551,14 @@ void ParseApi(const http_request_t& request, valhalla::Api& api) {
     throw valhalla_exception_t{101};
   };
 
+  // Sets the request document language parameter from the Accept-Language header
+  auto with_accept_language = [request](rapidjson::Document& document) {
+    auto language = ParseAcceptLanguage(request.headers);
+    auto& allocator = document.GetAllocator();
+
+    document.AddMember({"language", allocator}, {language, allocator}, allocator);
```

Comment:
rapidjson has an awful api.. `rapidjson::SetValueByPointer(doc, "/language", language)` is a little nicer to read IMO